packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -31,7 +30,6 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 - Added `Poller` utility singleton for handling continuous poll requests: see `Poller.addRequest, Poller.removeRequest` functions.
 - Added `pollData` optional function to `IZoweTree` class.
 - Added `ZosmfJesApi.downloadSingleSpool` optional api that can be used to download a single spool file in text or binary formats. [#2060](https://github.com/zowe/vscode-extension-for-zowe/issues/2060)
-- Added optional `id` variable to `IZoweTreeNode` interface, which can be used to designate a unique ID for a tree node. [#2215](https://github.com/zowe/vscode-extension-for-zowe/issues/2215)
 - Added new optional API, `IJes.downloadSingleSpool`, that can be used to download a single spool file in text or binary formats. [#2060](https://github.com/zowe/vscode-extension-for-zowe/issues/2060)
```

Comment:
Lines 32 and 33 appear to be duplicates

Reply:
After looking at these I believe they are different where the `IJes.downloadSingleSpool` is a new optional API for extenders and the line with `ZosmfJesApi.downloadSingleSpool` is the zOSMF adopting the new optional API.